photo/tests/test_mutations/test_contestSubmission.py

```diff
@@ -0,0 +1,140 @@
+from django.forms import ValidationError
+import factory
+import pytest
+from django.test import TestCase
+from django.utils import timezone
+import pytz
+from photo.fixtures import OUTDATED_SUBMISSION_ERROR_MESSAGE
+
+from photo.schema import schema
+from photo.tests.factories import (
+    ContestFactory,
+    ContestSubmissionFactory,
+    PictureFactory,
+    UserFactory,
+)
+from .graphql_mutations import (
+    contest_submission_creation_mutation,
+    contest_submission_update_mutation,
+    contest_submission_vote_mutation,
+)
+
+
+class ContestSubmissionTest(TestCase):
+    def setUp(self):
+        self.picture = PictureFactory()
+        self.contest = ContestFactory()
+
+    @pytest.mark.asyncio
+    async def test_create(self):
+        contest_submission = {
+            "picture": self.picture.id,
+            "contest": self.contest.id,
+        }
+
+        result = await schema.execute(
+            contest_submission_creation_mutation,
+            variable_values={"contestSubmission": contest_submission},
+        )
+
+        self.assertEqual(result.errors, None)
+        self.assertEqual(
+            result.data["create_contestSubmission"]["picture"]["user"]["email"],
+            self.picture.user.email,
+        )
+        self.assertEqual(
+            result.data["create_contestSubmission"]["picture"]["id"],
+            self.picture.id,
+        )
+        self.assertEqual(
+            result.data["create_contestSubmission"]["contest"]["id"], self.contest.id
+        )
+
+    def test_vote(self):
+        contest_submission = ContestSubmissionFactory()
+        user_vote = UserFactory()
+        result = schema.execute_sync(
+            contest_submission_vote_mutation,
+            variable_values={
+                "contestSubmission": contest_submission.id,
+                "user": str(user_vote.id),
+            },
+        )
+        self.assertEqual(result.errors, None)
+        self.assertEqual(
+            result.data["contest_submission_add_vote"]["results"]["votes"][0]["email"],
+            user_vote.email,
+        )
+
+    def test_update(self):
+        contest = ContestFactory()
+
+        user = UserFactory()
+        original_picture = PictureFactory(user=user, file="www.original.com")
+        newContestSubmission = ContestSubmissionFactory(
+            contest=contest, picture=original_picture
+        )
+
+        self.assertEqual(newContestSubmission.picture.file, original_picture.file)
+
+        newPicture = PictureFactory(user=user, file="www.test.com")
```

Comment:
why is the file set as url?

Reply:
this was how it was set in the previous tests. I'll change it according to the most recent model